### ISSUE TITLE
Unable to use docker-compose / docker compose with colima

### ISSUE THREAD

**Issue by sorakiu** [abiosoft/colima#265]
Repro steps

1. uninstall docker desktop using brew
2. install colima / docker / docker-compose / kubectl with brew install
3. manually run the following commands as recommended by brew install
```
  mkdir -p ~/.docker/cli-plugins
  ln -sfn /usr/local/opt/docker-compose/bin/docker-compose ~/.docker/cli-plugins/docker-compose
```
4. cd to a folder with a docker-compose.yaml file and run `docker-compose up`

I get the following error:
```
 docker compose up portal-db                                                                                                                                                                                          1 ↵
[+] Running 0/0
 ⠋ portal-db Pulling                                                                                                                                                                                                    0.0s
error getting credentials - err: exec: "docker-credential-desktop": executable file not found in $PATH, out: ``
```

I see some stuff on SO about what to do if this happens with Docker Desktop - but I wasn't sure what I should do with colima?

Really excited about this project!

**Comment by sorakiu** [comment]
I tried the solution in this article: https://forums.docker.com/t/docker-credential-desktop-exe-executable-file-not-found-in-path-using-wsl2/100225/4
and it started working.

**Comment by sorakiu** [comment]
i.e. changing CredsStore to CredStore in ~/.docker/config.json

**Comment by lazerg** [comment]
> i.e. changing CredsStore to CredStore in ~/.docker/config.json

Thank you, works like a charm

**Comment by rfay** [comment]
The reality is... Just delete ~/.docker.

**Comment by Defman21** [comment]
A better solution is: https://stackoverflow.com/a/72888813/3307167

TLDR: `brew install docker-credential-helper` and set `credsStore` to `osxkeychain`. With this setup, your credentials for docker hubs (e.g. hub.docker.com or your companies' private hub) won't be stored in plaintext but in your Keychain.

**Comment by wlhong-allo** [comment]
> i.e. changing CredsStore to CredStore in ~/.docker/config.json

Worked like a charm! thanks!

**Comment by jorgenbs** [comment]
In my case the command was incorrectly linked. Check `ls -l ~/.docker/cli-plugins/docker-compose` points correctly to `docker-compose`
